AutoCollection/diagnostic-channel/postgres.sub.ts

```diff
@@ -13,10 +13,11 @@ export const subscriber = (event: IStandardEvent<pg.IPostgresData>) => {
         const sql = (q.preparable && q.preparable.text) || q.plan || q.text || "unknown query";
         const success = !event.data.error;
         const conn = `${event.data.database.host}:${event.data.database.port}`;
+        const name = sql.length > 1024 ? sql.slice(0, 1021) + '...' : sql;
```

Comment:
This can be moved to a more generic location here:
https://github.com/microsoft/ApplicationInsights-node.js/blob/34a23fc9c869ecdd3b800d7748cf5ee81db6411f/Library/EnvelopeFactory.ts#L116-L118

Reply:
Thanks, I have moved it.